### DISCUSSION TITLE
Suggestion: Option to remove bottom toolbar and browser border

### DISCUSSION THREAD

**Discussion by mickeyeng** [lencx/Noi#94] [Ideas]
Love using the Noi; however, to keep it more minimal, could you please add an option to remove the browser border and bottom toolbar?

**Comment by lencx** [comment]
When the window border is entirely removed, users lose the ability to drag, maximize, or close the window using standard GUI controls. To compensate, alternative methods such as keyboard shortcuts must be introduced. This change can lead to significant usability issues, potentially causing unexpected complications for users.
